ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/tabularBlock/tabularBlock.component.html

```diff
@@ -1,65 +1,24 @@
 <h3>{{block.title}}</h3>
 
-                                <!-- to have different styles for composites, in reality will check something else -->
-<div class="tabular-container" [class.composite]="block.id === 'TABULAR_ID_2'">
-    <div class="headers" [gdColumns]="gridSettings.gridTemplateColumns">
-        <div *ngFor="let header of headers"
-             class="header cell"
-             [gdColumn]="header.gridColumn">
-            {{header.label}}
-        </div>
+<div class="tabular-container" [gdColumns]="gridSettings.gridTemplateColumns">
+    <div *ngFor="let header of headers" class="header cell" [gdColumn]="header.gridColumn" style="border: 1px solid grey;">
+        {{header.label}}
     </div>
```

Comment:
The headers (need to get rid of the style. My CSS is horrendous!

Reply:
Fixed in 0962fc142415b1180263bf165bef0fbc414b360f